webpack/loaders.js

```diff
@@ -37,7 +37,6 @@ const CSSLoader = {
         sourceMap: true,
         sassOptions: {
           importer: globImporter(),
-          outputStyle: 'compressed',
```

Comment:
but wouldn't we want this to be available when running webpack.prod.js? it uses the same loaders.

Reply:
Npm run build uses the webpack.prod.js
Npm run develop uses webpack.dev.js

webpack 4 will automatically minify if the mode is production. If the mode is development the css and js files will not be minified.